lib/bridge.js

```diff
@@ -101,7 +101,7 @@ Bridge.prototype.get = function get (nukiId, doNotValidate) {
   var self = this;
 
   if (doNotValidate) {
-    return Promise.resolve(new Nuki(self, nukiId, {delayBetweenRequests: self.delayBetweenRequests}));
+    return Promise.resolve(new Nuki(self, nukiId, nuki.deviceType || 0));
```

Comment:
`nuki` is not defined here

Reply:
True. Means we could either
1. remove `doNotValidate` and retrieve the `deviceType` in the course of the validation
OR
2. we add an additional parameter `deviceType`.

I would prefer the first option. What do you think?